examples/blog-with-auth/.prettierrc.js

```
@@ -0,0 +1,5 @@
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)